src/renderer/OpenGL.zig

```diff
@@ -302,6 +327,9 @@ pub const DerivedConfig = struct {
         // Copy our shaders
         const custom_shaders = try config.@"custom-shader".clone(alloc);
 
+        // Copy our background image
+        const background_image = try config.@"background-image".?.clone(alloc);
```

Comment:
Why is this using `.?`? Looks like it will just crash whenever the user doesn't specify a background image

```suggestion
        const background_image = if (config.@"background-image") |v| try v.clone(alloc) else null;
```

Reply:
Thanks for catching this.